src/qcodes/instrument_drivers/rigol/Rigol_DG4000.py

```diff
@@ -238,13 +241,15 @@ def __init__(
                 ch + "output_polarity",
                 get_cmd=output + "POL?",
                 set_cmd=output + "POL {}",
-                val_mapping={"normal": "NORM", "inverted": "INV"},
+                get_parser=_parse_string_strip,
+                val_mapping={"normal": "NORMAL", "inverted": "INVERTED"},
```

Comment:
@jenshnielsen should I update the val_mapping according to the instrument manual?

Reply:
I am not super familiar with this instrument but if its like most it supports both writing the short and the long form. I would say that if the long form is what is documented in the manual we should use that

Reply:
yes，I take the value from latest document of rigol DG4000

Reply:
It's setting value and return value are not the same.
![image](https://github.com/user-attachments/assets/d83f6131-878f-4460-845c-0687bf2b3dca)

set INV
return INVERTED.

Reply:
maybe we can wait until I confirm it again

Reply:
Usually when docs are written such as `INVerted` it meas that `INV` is the short form and `intervted` is the long form. I think its normally case insensitive but lets use what the instrument returns. Can you check that?

Reply:
![image](https://github.com/user-attachments/assets/36d44e91-124b-4e5b-845f-6f4901ef06c4)
![image](https://github.com/user-attachments/assets/8990cb62-3798-4666-a33b-830276d01f4b)
confirmed.

Reply:
@jenshnielsen good to go 👍